mill-scalafix/src/com/goyeau/mill/scalafix/ScalafixModule.scala

```diff
@@ -15,6 +15,23 @@ import scala.jdk.CollectionConverters.*
 trait ScalafixModule extends ScalaModule {
   def scalafixConfig: T[Option[os.Path]] = T(None)
   def scalafixIvyDeps: T[Agg[Dep]]       = Agg.empty[Dep]
+
+  /** Override this to filter out repositories that don't need to be passed to scalafix
+    *
+    * Repositories passed to scalafix need to be converted to the coursier-interface API (coursierapi.*). This can be an
+    * issue for non-Maven or Ivy repositories. Overriding this task and filtering those repositories out allows to work
+    * around that.
+    */
+  def scalafixRepositories: Task[Seq[Repository]] = Task.Anon {
+    repositoriesTask().filter {
+      case repo if repo.getClass.getName == "mill.scalalib.JavaModule$InternalRepo" =>
+        // Change to this when bumping to Mill > 0.12.5:
+        // case _: mill.scalalib.JavaModule.InternalRepo =>
+        // no need to pass Mill's internal repository to scalafix
+        false
+      case _ => true
+    }
+  }
```

Comment:
nit: Do you mind adding a space after this line?

Reply:
Sure